src/parser.mjs

```diff
@@ -682,7 +689,8 @@ function reviveDate(string) {
 		return null
 	string = string.trim()
 	var [dateString, timeString] = string.split(' ')
-	var [year, month, day] = dateString.split(':').map(Number)
+	var [year, month, day] = dateString.split(/[:\.]/).map(Number)
+    if (day > 1900) [year, day] = [day, year]
```

Comment:
The Wiris 2 uses a day.month.year format instead of the ISO year:month:day. The check might be a bit dirty, but it should work in all real-life use-cases I think... 😛